activiti-core/activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/CollaborationExtensionElementsConverterTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2010-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.editor.language.xml;
+
+import org.activiti.bpmn.model.BpmnModel;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class CollaborationExtensionElementsConverterTest extends AbstractConverterTest {
+
+  @Test
+  public void convertXMLToModel() throws Exception {
+    BpmnModel bpmnModel = readXMLFile();
+    Assertions.assertNotNull(bpmnModel);
```

Comment:
Would you mind adding some extra checks to the converted model (at least to assert that pools `Pool` and `Pool2`) are present?

Reply:
@erdemedeiros will do it now.

Reply:
@erdemedeiros done. running ci.